kqueen/engines/gce.py

```diff
@@ -61,7 +61,10 @@ class GceEngine(BaseEngine):
                     'required': True,
                     'min': 1,
                     'number': True
-                }
+                },
+                'help_message':
+                    'To enable network policy, your cluster must have at least 2 nodes. '
+                    'The recommended minimum size cluster to run network policy enforcement is 3 Standard-1 instances'
```

Comment:
I would rephrase to something like:
To run the network policy enforcement, the cluster must consist of at least 3 ...
Or:
The recommended minimum cluster size to run the network policy enforcement is ...